yt_dlp/extractor/facebook.py

```diff
@@ -678,6 +678,13 @@ def parse_attachment(attachment, key='media'):
                 # honor precise duration in video info
                 if video_info.get('duration'):
                     webpage_info['duration'] = video_info['duration']
+                # preserve preferred_thumbnail in video info
+                if video_info.get('thumbnail'):
+                    if not [x for x in webpage_info['thumbnails'] if video_info['thumbnail'].split('?')[0] in x['url']]:
+                        webpage_info['thumbnails'].append({
+                            'url': video_info['thumbnail'],
+                            'preference': 1
+                        })
```

Comment:
After visually comparing the `webpage_info` thumbnail to the `video_info` thumbnail, I feel that the `webpage_info` thumbnail is worthless and does not need to be kept if we have the `video_info` thumb. Seeing as the facebook extractor is already way too complicated, I think we should simplify this as much as possible and just overwrite the `webpage_info` thumb:
```suggestion
                if video_info.get('thumbnail')
                    webpage_info['thumbnail'] = video_info['thumbnail']
```